src/utils/Utility.ts

```diff
@@ -42,7 +43,7 @@ export const getSrc = ({ urlEndpoint, lqip, src, path, transformation, transform
     urlEndpoint: urlEndpoint || contextOptions.urlEndpoint,
     src: src || contextOptions.src || undefined,
     path: path || contextOptions.path || undefined,
-    transformation: transformation || contextOptions.transformation,
+    transformation: (transformation || contextOptions.transformation) as Array<Transformation>,
```

Comment:
This is fine, but can we not improve the propTypes of `transformation` and `contextOptions.transformation`?

Reply:
done